redis_docker.conf

```diff
@@ -0,0 +1,9 @@
+# NOTE: This file is used within the Docker setup for local development.
+# Modify the `redis.conf` file when we permanently switch to Docker.
```

Comment:
Please make this a TODO statement and open an issue.

Reply:
done

Reply:
confirmed this is done